### PR TITLE
fix(assets): display bpmn-element images

### PR DESCRIPTION
**Pull request by timstawowski**
## Description

Images of bpmn-elements couldn't be displayed because of sized image tags.

**Before**
missing-bpmn-elements

**After**
with-bpmn-elements

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [X] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [X] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [X] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
I don't think our `deploy` label workflow will work for externals. We'll need to review this one by running locally @christinaausley, but it was a good idea to try to add it!

File: docs/components/best-practices/development/understanding-transaction-handling-c7.md [hunk 1]
```diff
@@ -92,27 +92,27 @@ Aside a general strategy to mark service tasks as being save points you will oft
 
 - _User tasks_ <img src="/img/bpmn-elements/task-user.svg" width="40"/>: This savepoint allows users to complete their tasks without waiting for expensive subsequent steps and without seeing an unexpected rollback of their user transaction to the waitstate before the user task. Sometimes, e.g. when validating user input by means of a subsequent step, you want exactly that: rolling back the user transaction to the user task waitstate. In that case you might want to introduce a savepoint right after the validation step.
```
Comment (akeller) [inline docs/components/best-practices/development/understanding-transaction-handling-c7.md:93]: image

This one is also not visible on the page.

**Review by akeller — CHANGES_REQUESTED** [review]
First, I want to thank you for your attention to detail and tenacity in trying to resolve this! 

Second, while removing the width does make the images visible, the experience is still not great. 

image

We are incremental in our delivery and shipping, so I would accept this PR if all the images are appearing, even if the experience is not so great. We can adjust that in a separate PR.

Your PR also showed me some other issues with how our BPMN.io images/diagrams are rendering. They are visible but cut off. I will create a backlog item for this.

File: docs/components/best-practices/development/understanding-transaction-handling-c7.md (hunk 1 above)
Comment (timstawowski) [reply docs/components/best-practices/development/understanding-transaction-handling-c7.md]: added the missing ones.

**Review by akeller — APPROVED** [review]
Thank you!

**Pull request merged**